include/vara/Feature/Feature.h

```diff
@@ -1,26 +1,143 @@
-#ifndef VARA_FEATURE_FEATURE_H
-#define VARA_FEATURE_FEATURE_H
+#ifndef VARA_FEATUREMODEL_FMFEATURE_H
+#define VARA_FEATUREMODEL_FMFEATURE_H
 
 #include "llvm/ADT/SmallVector.h"
+#include "llvm/Support/raw_ostream.h"
+#include <variant>
 
-namespace vara::feature {
+#include "vara/Feature/Relationship.h"
 
-class Feature {
-  llvm::SmallVector<int, 8> Data;
+using std::string;
+namespace vara {
+
+//===----------------------------------------------------------------------===//
+//                               Feature Class
+//===----------------------------------------------------------------------===//
+
+/// \brief Base class for components of \a FeatureModel.
+class FMFeature {
```

Comment:
We still call this Feature, with the correct namespace there will not be a name clash